airflow/models/taskinstance.py

```diff
@@ -715,6 +716,13 @@ def _execute_task(task_instance: TaskInstance | TaskInstancePydantic, context: C
             if not task_instance.next_kwargs:
                 task_instance.next_kwargs = {}
             task_instance.next_kwargs[f"{task_to_execute.__class__.__name__}__sentinel"] = _sentinel
+        elif task_instance.next_method == "__trigger_exit__":
```

Comment:
Where does this "__trigger_exit__" come from? I thought it was a constant. Even if we move it out of the `TaskDeferred`, I think we probably still should make it a constant for op author to use. Also this behavior seems to be missed in the doc

Reply:
The next_method doesn't matter in ending task. It can be None as well. But I can create a constant as mentioned.

Reply:
Removed it.